linux/datadog_wrapper

```diff
@@ -133,10 +133,18 @@ setUpDotnetEnv() {
         return
     fi
 
+    echo "Exporting environment variables"
     export CORECLR_ENABLE_PROFILING=1
     export CORECLR_PROFILER="{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
     export CORECLR_PROFILER_PATH="${DD_DIR}/Datadog.Trace.ClrProfiler.Native.so"
     export DD_DOTNET_TRACER_HOME="${DD_DIR}"
+
+    if [[ "${DD_PROFILING_ENABLED:-,,}" == "1" ]]; then
+        echo "Profiler is enabled."
+        export LD_PRELOAD="${DD_DIR}/continuousprofiler/Datadog.Linux.ApiWrapper.x64.so"
+    fi
+    echo "Creating log folder for .NET APM"
+    ${DD_DIR}/createLogPath.sh
```

Comment:
I'm kind of surprised we weren't doing this already 😅 we may need to make this executable first?

Alternatively, I _think_ you can run it as (but don't quote me):

```suggestion
    sh ${DD_DIR}/createLogPath.sh
```